pipeline.py

```diff
@@ -70,20 +70,36 @@
     description="InstructLab pipeline",
 )
 def ilab_pipeline(
+    # Model I/O
+    input_model_uri: str,
+    output_oci_model_uri: str,
+    output_oci_registry_secret: str,
+    output_model_name: str,
+    output_model_version: str,
+    output_model_registry_name: str,
```

Comment:
I think this should make them all optional as we agreed on since the user can just let the trained model be uploaded to S3.
```suggestion
    output_oci_model_uri: str = None,
    output_oci_registry_secret: str = None,
    output_model_name: str = None,
    output_model_version: str = None,
    output_model_registry_name: str = None,
```